### ISSUE TITLE
Loading Spinner iOS

### ISSUE THREAD

**Issue by ghammond100** [arielfaur/ionic-audio#11]
Hi, this project is just what I'm looking for so thanks!

On iOS (iPhone 6) the loading spinner appears to freeze. I'm using it to stream mp3's from the web. The spinner spins once or twice and then stops until the audio begins to play about 30 seconds later. On my android test device (galaxy S3) the spinner spins and audio begins to play after about 1 or 2 seconds.

**Comment by arielfaur** [comment]
Hmmm, I have an Android device too but mine seems to work as expected. How large are your audio files? Are your files hosted on Amazon or similar?

**Comment by ghammond100** [comment]
All files are hosted on Amazon S3 and about 20mb. Seems to work good on Android, it's iOS where I have the delay and the spinner freezes.

**Comment by arielfaur** [comment]
This module uses the Cordova Media plugin for playback. Apparently it does not support streaming of MP3 files, unless you use an M3U playlis I was told.  What type of URLs are you using? And are you using Cloudfront or Amazon Elastic Transcoder?

**Comment by ghammond100** [comment]
I'm just using a url like http://bucket.s3.amazonaws.com/path/to/my.mp3". I'm not using cloudfront or Amazon Elastic Transcoder.

**Comment by arielfaur** [comment]
I think the plugin will try to download the whole file before playback, so that may be the lag you are experiencing. I bet you can check that with Chrome Dev tools in the network tab.

**Comment by ghammond100** [comment]
Okay I'll look into that. Thank you for the info.

**Comment by ghammond100** [comment]
I created a m3u file with a single entry pointing to a mp3 on Amazon S3. I tested it with VLC media player to make sure it worked. Unfortunately it doesn't seem to work with this plugin. If I replace the track mp3 URL with the m3u URL nothing happens. The loading spinner just spins.

I found you forum discussion with ghenry22 that mentions the m3u. I can't seem to find any other information about m3u and Cordova media plugin. Did you incorporated ghenry22's fork of the Cordova Media plugin?

**Comment by ghammond100** [comment]
I remembered the Cordova Media Plugin is a dependency so I removed the official plugin and installed ghenry22's fork. I still was unable to use a m3u file. I tested on iOS and android. On iOS the error I see is:
    `Failed to initialize AVAudioPlayer: The operation couldn’t be completed. (OSStatus error 1954115647.)`

After searching this error I believe it means its an unsupported file format for AVAudioPlayer which I believe is what Cordova Media uses to play audio on iOS. 

By the way, while I was changing versions of Cordova Media I noticed your documentation lists the following way to install it:
`ionic plugin add org.apache.cordova.media`

This gives a warning that to get the latest release it should be installed like:
`ionic plugin add cordova-plugin-media`
